aries-site/src/examples/components/menu/MenuExample.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Box, Menu } from 'grommet';
+
+export const MenuExample = () => {
+  const items = [
+    { label: 'Change username' },
+    { label: 'Reset Password' },
+    { label: 'Logout' },
+  ];
+
+  return (
+    <Box round="xsmall">
+      <Menu label="Account Information" items={items} width="medium" />
```

Comment:
Trying to think of ways to reinforce the "action" oriented nature of Menu with a label containing a verb...

```suggestion
      <Menu label="Manage Account" items={items} width="medium" />
```

```suggestion
      <Menu label="Manage User" items={items} width="medium" />
```

```suggestion
      <Menu label="Account Actions" items={items} width="medium" />
```